addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1159,6 +1160,10 @@ var AbstractFieldBinary = AbstractField.extend({
 });
 
 var FieldBinaryImage = AbstractFieldBinary.extend({
+    fieldDependencies: _.extend({}, AbstractFieldBinary.prototype.fieldDependencies, {
```

Comment:
you should also add the fieldDependencies attribute to AbstractField (and document it)